charts/kubeai/values.yaml

```diff
@@ -5,6 +5,9 @@
 secrets:
   huggingface:
     token: ""
+    # secretName is used to pass the huggingface token value via an existing k8s secret
+    # the value is pulled from the key, "token".
+    secretName: ""
```

Comment:
To keep things consistent let's follow the same convention of using `.create` and `.name` as is used for `.serviceAccount` lower down in the values file.

(This convention was templated by `helm create`)

```yaml
serviceAccount:
  # Specifies whether a service account should be created
  create: true
  # Automatically mount a ServiceAccount's API credentials?
  automount: true
  # Annotations to add to the service account
  annotations: {}
  # The name of the service account to use.
  # If not set and create is true, a name is generated using the fullname template
  name: ""
```